<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QDialog" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>851</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bedNum">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="numBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>根据ID号搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nameBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>根据姓名搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showAll">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>显示全部病人</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
